### PR TITLE
updated the twitter logo

### LINKED ISSUE #8994: Twitter logo needs to be updated to X
### Description
in the footer of [docs.meshery.io](https://docs.meshery.io/) twitter's old logo is needs to be updated to X

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by joshijoe05**
Fixes #8994 
Updated the twitter logo.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 42fa9def7dc584df32eea0e3996b4aa008e2e4b6 at: https://651af3ff7d16d7171cc565df--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 36ac3ad47f25fadab8e92a050dcf16b46dabf4e3 at: https://651af8140c4dd14c4c9b545d--meshery-docs-preview.netlify.app

**Comment by joshijoe05** [comment]
twitter
This was the updated twitter logo

File: docs/_includes/footer.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
...
+          src = "https://about.twitter.com/content/dam/about-twitter/x/brand-toolkit/logo-black.png.twimg.1920.png"
```
Comment (leecalcote) [inline docs/_includes/footer.html:80]: Thanks! Let's avoid using any external image references. All image references should be internal to the site. Please include a copy of the new Twitter logo in your PR.

Also, please use and SVG, not a PNG.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_includes/footer.html (hunk 1 above)
Comment (joshijoe05) [reply docs/_includes/footer.html:80]: Can you mention where to find internal references?

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by leecalcote** [comment]
@joshijoe05 are you going to take care of this - https://github.com/meshery/meshery/pull/9001#issuecomment-1743667102?

**Comment by joshijoe05** [comment]
@leecalcote Yeah I'll complete this issue ASAP!

**Pull request closed without merging**